### ISSUE TITLE
Potential accessibility fixes RWD: Learn Accessibility by Building a Quiz

### ISSUE THREAD

**Issue by bbsmooth** [freeCodeCamp/freeCodeCamp#47830]
### Describe the Issue

The following is a list of accessibility issues ranked by severity. I don't expect that all of these will be implemented. This is more of a "in a perfect world" scenario (especially toward the bottom of the list). A few of these are complex enough that if we decide to fix them we should probably create new individual issues for them so that we can discuss them more.

**Some things that should probably be fixed:**

```[tasklist]
### MUST FIX
- [x] The minimum width on the logo is `100px` which makes it very hard to read. I think that needs to be made much higher. Possibly use `rem` for the width so it scales with the other text. - #54824
- [ ] Header CSS needs to be reworked to use media queries to rearrange its content for narrower view ports (instead of playing with font sizes and image widths)
- [x] The `Question #1` and `Question #2` text look like headings and should probably be marked up as headings.
- [x] Steps 23-25 are problematic. Instead of using D.O.B., we should use "Date of Birth". - #54829
```

```[tasklist]
### Should Fix
- [x] Get rid of the placeholder text in the `textarea`. The question text is self-explanatory on its own so the placeholder text is not adding any value. https://github.com/freeCodeCamp/freeCodeCamp/pull/55739
- [ ] "Question 1" and "Question 2" sr-only text be added to the beginning of their respective questions so that screen reader users navigating by tab stop will hear the question number as well as the question text.
- [x] Don't use `px` for font sizing. Use `rem` for all font sizes in order to respect the default font size the user has set in their browser.
```

```[tasklist]
### NICE TO FIX
- [ ] Move the `h1` element out of the `header` and make it the first element in `main`. Best practices would say that the `h1` element should be the first element in `main` (but this is not an accessibility requirement). WIP: #57944 
- [x] Remove fixed positioning from header. - #58002
- [x] Get rid of Step 68. The `accesskey` attribute is hardly ever used and has a bunch of accessibility issues.
```

### Affected Page

A whole ton of them.

### Your code

N/A

### Expected behavior

Go through the list and decide which ones should be fixed. Create separate issues as needed for more complex issues.

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

**Comment by evelynmitchell** [comment]
In 14-responsive-web-design-22/learn-accessibility-by-building-a-quiz) the screen reader text for date of birth is enclosed in parenthesis. This may not be as helpful to someone using a screen reader, as the phrase without the parenthesis. 

s/(Date of Birth)/Date of Birth/

**Comment by naomi-lgbt** [comment]
This has been sitting here without much discussion. Let's go ahead and open it up for contribution.

To facilitate the review process, please create individual PRs for the tasks outlined above.

**Comment by Supravisor** [comment]
Have a block regarding: Steps 23-25 are problematic. Instead of using D.O.B., we should use "Date of Birth" 

https://github.com/freeCodeCamp/freeCodeCamp/pull/54829#discussion_r1603669716

> > Although the `Date of Birth` text is descriptive, it can be problematic for visually impaired users.
> 
> I don't see why "Date of Birth" can be problematic for visually impaired users. And changing the label to spell out "Date of Birth" means that the `sr-only` text is not needed, otherwise non-sighted users would hear "Date of Birth" twice.
> 
> The "D.O.B" text change needs to be accompanied with an alternative example/exercise that makes use of the `sr-only` class. So I'd suggest going back to the issue thread and start a discussion on how the lessons should be updated.

Any ideas on how to incorporate the `sr-only` class for this lesson?

**Comment by bbsmooth** [comment]
> Any ideas on how to incorporate the `sr-only` class for this lesson?

The first item under "Should Fix":

> "Question 1" and "Question 2" sr-only text be added to the beginning of their respective questions so that screen reader users navigating by tab stop will hear the question number as well as the question text.

**Comment by YT-Tux** [comment]
It is January 9 2025 and I found this project open for the capabilities I have. Is there anything I can do to help with this?

What is the progress?

**Comment by ilenia-magoni** [comment]
@YT-Tux the checkpoints above seem updated to current state of the project

**Comment by YT-Tux** [comment]
Ok

**Comment by ilenia-magoni** [comment]
> Is there anything I can do to help with this?

You can open a PR to contribute some of the changes. note that @Supravisor has a PR #57944 open that touches many steps here, without coordination there can be quite a lot of conflicts

**Comment by bbsmooth** [comment]
I am officially withdrawing my recommendation to move the h1 into the main element. While it is true that the h1 is usually in the main element where the main content begins, it is not a WCAG requirement and it makes sense to have it in the header for some designs, such as the one being used for this quiz. I think I was being a little overzealous with this recommendation and I fear I have created unnecessary concern over something that doesn't need to be fixed. 

The h1 is perfectly fine where it is.

**Comment by hbar1st** [comment]
should someone edit the list of Nice to Fix to remove the h1 recommendation then?

**Comment by majestic-owl448** [comment]
I propose closing this, unless there is a reason not to
